dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -1020,7 +1020,7 @@ shared_ptr<BusInterface>
 DataInterfaceIIDM::findCalculatedBusInterface(const string& voltageLevelId, const int& node) {
   map<string, vector<shared_ptr<CalculatedBusInterfaceIIDM> > >::iterator iter = calculatedBusComponents_.find(voltageLevelId);
   if (iter == calculatedBusComponents_.end())
-    throw DYNError(Error::MODELER, UnknownCalculatedBus, voltageLevelId);
+    throw DYNError(Error::MODELER, UnknownVoltageLevel, voltageLevelId);
```

Comment:
When I read it again it is really a calculatedBus that is not found, so maybe we should create a new error message without a node

Reply:
I'll reuse `UnknownCalculatedBusWithBbs` since there is a `bbsId`